modules/rpc/yarpc.go

```diff
@@ -183,3 +191,18 @@ func (m *YarpcModule) IsRunning() bool {
 	defer m.stateMu.RUnlock()
 	return m.rpc != nil
 }
+
+type yarpcDispatcherFn func(yarpc.Config) (yarpc.Dispatcher, error)
+
+// RegisterDispatcher allows you to override the YARPC dispatcher registration
+func RegisterDispatcher(dispatchFn yarpcDispatcherFn) {
+	_dispatcherFn = dispatchFn
```

Comment:
It will always be non-nil, since we call Register with the default dispatcher on init